File: desafio.backend/desafio.backend/Models/TaskModel.cs
using desafio.backend.Entities.Enums;
using System.Collections;
using System.Collections.Generic;

namespace desafio.backend.Models
{
    public class TaskModel
    {
        public string Id { get; set; }
        public string genratorId { get; set; }
        public string GeneratorName { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string Responsible { get; set; }
        public string ResponsibleName { get; set; }
        public ICollection<ActivityModel> Activitys { get; set; }
        public EStatus Status { get; set; }
       
    }
}

File: desafioFrontend/src/app/shared/models/Task.model.ts
import { Status } from 'src/app/shared/enums/Status.enum';

export class Task {
  Generator?: string;
  Title?: string;
  Description: string;
  Responsible: string;
  Activity?: string;
  Status: Status;
  GeneratorName: string;
  ResponsibleName: string;
}
